Configuration/Settings.yaml

```diff
@@ -51,11 +51,11 @@ Neos:
               component: Wysiwyg\CookieHandling\Domain\Http\CookieCleanupComponent
 
             cookieConsentHandler:
```

Comment:
Just a minor thing, but could you bring the three process blocks in the chain into the right order? Makes it easier to read when they are in the order they happen in.